build.gradle

```diff
@@ -13,7 +13,7 @@ repositories {
 }
 
 dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'junit', name: 'junit', version: '4.4'
```

Comment:
Any reason to drop back to 4.4 from 4.12?

Reply:
This is a long story... 😅 . There shouldn't be problems if I get back to 4.12, I think